### PR TITLE
Update recommended app notifications

### PR DESCRIPTION
**Pull request by juliushaertl**
Implements #225 

image

### CODE REVIEW

File: img/apps/social.svg [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (rullzer) [inline img/apps/social.svg]: Minify those?

Comment (juliushaertl) [reply img/apps/social.svg]: Done

File: lib/Notification/Notifier.php [hunk 2]
```diff
@@ -166,10 +171,31 @@ protected function setAppHintDetails(INotification $notification, $languageCode,
...
+				$notification->setParsedMessage($l->t('Kanban style organization for personal planning and team projects.'));
+				$appLink = '/organization/deck';
+				break;
+			case 'notes':
+				$notification->setParsedSubject($l->t('App recommendation: Notes'));
+				$notification->setParsedMessage($l->t('Distraction-free notes and writing.'));
+				$appLink = '/organization/notes';
+				break;
+			case 'social':
+				$notification->setParsedSubject($l->t('App recommendation: Nextcloud Social'));
```
Comment (jancborchardt) [inline lib/Notification/Notifier.php]:
```suggestion
				$notification->setParsedSubject($l->t('App recommendation: Social'));
```

**Comment by jancborchardt** [comment]
Awesome! Besides cutting the "Nextcloud" from "Nextcloud Social" just one thing: Can we have the order reversed? That is, so it shows up for people as in the issue:
- Tasks
- Deck
- Notes
- Social
- Group folders

Thanks! :)

**Comment by juliushaertl** [comment]
@jancborchardt Reordered the list as wished :wink:

File: lib/Notification/Notifier.php (hunk 2 above)
Comment (jancborchardt) [reply lib/Notification/Notifier.php]: @juliushaertl did you see this one? :)

Comment (juliushaertl) [reply lib/Notification/Notifier.php]: Sorry missed that, pushed again.

**Pull request merged**